src/pages/SignUp.js

```diff
@@ -50,15 +51,30 @@ function SignUp() {
     return errors;
   };
 
+  const [selectedValue, setSelectedValue] = React.useState('researcher');
+  
+  const handleChange = (event) => {
+      setSelectedValue(event.target.value);
+    }
+
+  const navigate = useNavigate();
+
   const handleSubmit = (values) => {
-    console.log(values); // TODO: get radio buttons value as userType
     const { email, password, firstName, lastName, userType } = values;
-
+    
     try {
       Authentication.signUp(email, password, firstName, lastName, userType);
     } catch (error) {
-      // TODO: show the error message to user
+    // TODO: show the error message to user
+    }
+
+    if(selectedValue==="researcher"){
```

Comment:
nit: spacing `if (selectedValue === "researcher") {`

Reply:
also might be better if we have enum/number instead of comparing string, it's more efficient. this can be done in another PR tho :)